helm-charts/falcon-kac/templates/deployment_webhook.yaml

```diff
@@ -0,0 +1,360 @@
+          httpGet:
+            path: /startz-kac
+            port: {{ .Values.webhookPort }}
+            scheme: HTTPS
+          periodSeconds: 2
+          successThreshold: 1
+          timeoutSeconds: 1
+        volumeMounts:
+        - mountPath: /tmp
+          name: crowdstrike-falcon-vol0
+        - mountPath: /var/private/
+          name: crowdstrike-falcon-vol1
+      nodeSelector:
+        kubernetes.io/os: linux
+      priorityClassName: system-cluster-critical
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      shareProcessNamespace: true
+      {{- if .Values.tolerations }}
+      tolerations:
+      {{- with .Values.tolerations }}
+      {{- toYaml . | nindent 6 }}
+      {{- end }}
+      {{- end }}
+      volumes:
+      - name: {{ include "falcon-kac.name" . }}-tls-certs
+        secret:
+          secretName: {{ include "falcon-kac.name" . }}-tls
+      - name: crowdstrike-falcon-vol0
+        emptyDir:
+          sizeLimit: 256Mi
+      - name: crowdstrike-falcon-vol1
+        emptyDir:
+          sizeLimit: 4Ki
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: {{ include "falcon-kac.webhookName" . }}
+  labels:
+    {{- include "falcon-kac.labels" . | nindent 4 }}
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    caBundle: {{ $caCert }}
+    {{- if .Values.domainName -}}
+    url: https://{{ $webhookFullName }}:443/validate
+    {{- else }}
+    service:
+      name: webhook
+      namespace: {{ .Release.Namespace }}
+      path: /validate
+    {{- end }}
+  failurePolicy: {{ .Values.webhook.failurePolicy }}
+  matchPolicy: Equivalent
+  name: {{ include "falcon-kac.webhookName" . }}
+  namespaceSelector:
+    matchExpressions:
+    - key: kubernetes.io/metadata.name
+      operator: NotIn
+      values:
+      - {{ .Release.Namespace }}
+      - kube-system
+      - kube-public
+      - falcon-system
+      {{- if $myns }}
+      {{- range $ns := $myns }}
+      - {{ $ns }}
+      {{- end -}}
+      {{- end -}}
+      {{ include "falcon-kac.openshiftNamespaces" . | nindent 6 }}
+    - key: {{ include "falcon-kac.webhookName" . }}/admission-review
+      operator: NotIn
+      values:
+      - disabled
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - pods
+    - pods/ephemeralcontainers
+    scope: '*'
+  sideEffects: None
+  timeoutSeconds: 5
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    caBundle: {{ $caCert }}
+    {{- if .Values.domainName -}}
+    url: https://{{ $webhookFullName }}:443/validate
+    {{- else }}
+    service:
+      name: webhook
+      namespace: {{ .Release.Namespace }}
+      path: /validate
+    {{- end }}
+  failurePolicy: Ignore
+  matchPolicy: Equivalent
+  name: workload.{{ include "falcon-kac.webhookName" . }}
```

Comment:
```suggestion
  name: client.validating.admission.falcon.crowdstrike.com
```

Reply:
The 2nd webhook validates kubernetes [workloads](https://kubernetes.io/docs/concepts/workloads/) as opposed to pods which are validated by 1st webhook
The prefix 'client' does not seem appropriate here

Reply:
`workloads.validating.admission.falcon.crowdstrike.com` is also fine.